models/articles-model.js

```diff
@@ -10,7 +11,20 @@ exports.selectArticleById = (id) => {
     })
 }
 
-exports.selectAllArticles = () => {
+exports.selectAllArticles = (topic) => {
+    if (topic) {
+        // checkExists('topics', 'slug', topic)
+        // .then(() => {
```

Comment:
This comment can go since this logic is in the controller :)

Reply:
Oh I missed that one! By the way, I tried implementing it in the model, but it wasn't working and was somehow breaking other tests of different APIs